docs/settings.rst

```diff
@@ -4,14 +4,40 @@
 Settings
 ========
 
-``RATELIMIT_CACHE_PREFIX``:
-    An optional cache prefix for ratelimit keys (in addition to the
-    ``PREFIX`` value). *rl:*
-``RATELIMIT_ENABLE``:
-    Set to ``False`` to disable rate-limiting across the board. *True*
-``RATELIMIT_USE_CACHE``:
-    Which cache (from the ``CACHES`` dict) to use. *default*
-``RATELIMIT_VIEW``:
-    A view to use when a request is ratelimited, in conjunction with
-    ``RatelimitMiddleware``. (E.g.: ``'myapp.views.ratelimited'``.)
-    *None*
+``RATELIMIT_CACHE_PREFIX``
+--------------------------
+
+An optional cache prefix for ratelimit keys (in addition to the ``PREFIX``
+value defined on the cache backend). Defaults to ``'rl:'``.
+
+``RATELIMIT_ENABLE``
+--------------------
+
+Set to ``False`` to disable rate-limiting across the board. Defaults to
+``True``.
+
+May be useful during tests with Django's |override_settings|_ testing tool,
+for example:
+
+.. code-block:: python
+
+    from django.test import override_settings
+
+    with override_settings(RATELIMIT_ENABLE=False):
+        result = call_the_view()
+
+.. |override_settings| replace:: ``override_settings()``
+.. _override_settings: https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.override_settings.
```

Comment:
oh, interesting! i didn't know you could do this kind of combination

Reply:
🎩✨